addons/html_builder/static/src/core/move_plugin.js

```diff
@@ -116,6 +119,7 @@ export class MovePlugin extends Plugin {
         const buttons = [];
         this.overlayTarget = target;
         this.noScroll = this.overlayTarget.matches(this.noScrollSelector);
+        const reverseButtons = this.isEditableRTL !== this.isBackendRTL;
```

Comment:
It seems that with the backend in rtl and an rtl website, the arrows label are reversed (but they work correctly)
<img width="1437" height="477" alt="image" src="https://github.com/user-attachments/assets/cf1e2919-97a2-404c-8629-b293b539920b" />

Same with an english website on an rtl backend 
<img width="1326" height="463" alt="image" src="https://github.com/user-attachments/assets/1638ea56-2f2e-41ed-9555-500ea103795a" />